client_unix_test.go

```diff
@@ -2,12 +2,30 @@
 
 package containerd
 
+import (
+	"runtime"
+)
+
 const (
 	defaultRoot    = "/var/lib/containerd-test"
 	defaultAddress = "/run/containerd-test/containerd.sock"
-	testImage      = "docker.io/library/alpine:latest"
+)
+
+var (
+	testImage string
 )
 
 func platformTestSetup(client *Client) error {
 	return nil
 }
+
+func init() {
+	switch runtime.GOARCH {
+	case "amd64":
+		testImage = "docker.io/library/alpine:latest"
+	case "arm":
+		testImage = "docker.io/armhf/alpine:latest"
+	default:
+		testImage = "docker.io/" + runtime.GOARCH + "/alpine:latest"
```

Comment:
/me goes into deep depression realizing that @crosbymichael hasn't been following multi-arch work in the Docker world at all! 😝 

Given the architecture scope of building containerd is pretty much limited to those architectures, I think this is fine other than maybe a comment about which arch. namespaces exist? I don't know what FreeBSD or Solaris do for images, but they would be outside of this test framework already?